flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/sink/BatchExecutionFileSinkITCase.java

```diff
@@ -74,7 +74,8 @@ protected JobGraph createJobGraph(String path) {
                 .map(new BatchExecutionOnceFailingMap(NUM_RECORDS, triggerFailover))
                 .setParallelism(NUM_SINKS)
                 .sinkTo(createFileSink(path))
-                .setParallelism(NUM_SINKS);
+                .setParallelism(NUM_SINKS)
+                .uid("sink");
```

Comment:
I think we'd better to set uid in the compaction related tests to also tests normal sinks works without uid.